toolchains/sysimage/run_in_namespace_test.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+run_in_namespace=$(realpath toolchains/sysimage/run_in_namespace.sh)
+ubuntu_base=$(realpath external/ubuntu-base-24.04.1-base-amd64.tar.gz/file/downloaded)
```

Comment:
can you pass the path to the archive as an environment variable? otherwise we'll get obscure errors whenever the path or filename changes

Reply:
Done